README.md

```diff
@@ -13,42 +13,46 @@ JWST Calibration Pipeline
 Installation
 ------------
 
-The ``jwst`` package can be installed into a virtualenv or conda environment via ``pip``.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
+The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
 
-    conda create -n jwst_env python=3.7
+    conda create -n jwst_env python
     conda activate jwst_env
 
 ### Installing for end-users ###
 
-To install a released (tagged) version, you can install directly from Github.  To install tagged release ``jwst 0.14.2``:
+To install a released (tagged) version, you can install directly from Github.  To install release `jwst 0.15.0`:
 
-    pip install git+https://github.com/spacetelescope/jwst@0.14.2
+    pip install git+https://github.com/spacetelescope/jwst@0.15.0
 
-The latest development version (from ``master``) can also be installed from Github:
+The latest development version (from `master`) can also be installed from Github:
 
     pip install git+https://github.com/spacetelescope/jwst
 
 As can a particular commit hash:
 
     pip install git+https://github.com/spacetelescope/jwst@3f03323c
 
+To include extra testing dependencies, the syntax is as follows:
+
+    pip install "jwst[test] @ git+https://github.com/spacetelescope/jwst"
```

Comment:
Is the whitespace in the middle of "@ git" intentional?

Reply:
Yes.  It is needed.  And also the quotes.